tests/examples/test_wds_e2e.py

```diff
@@ -96,35 +99,45 @@ def test_wds(catalog, webdataset_tars):
 
 
 def test_wds_merge_with_parquet_meta(catalog, webdataset_tars, webdataset_metadata):
-    wds = DataChain.from_storage(Path(webdataset_tars).as_uri()).gen(
-        laion=process_webdataset(spec=WDSLaion), params="file"
-    )
+    with warnings.catch_warnings():
```

Comment:
should we also put a limit of some sort to the ClickHouse-SQLAlchemy? so that we can remove this code later 

also, worth mentioning the version at list where it is still the case

(not a blocker to the PR)